inference-engine/src/transformations/src/transformations/softplus_fusion.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/softplus_fusion.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+ngraph::pass::SoftPlusFusion::SoftPlusFusion() {
+    auto input = ngraph::pattern::any_input();
+    auto exp = std::make_shared<ngraph::opset4::Exp>(input);
+    auto add = std::make_shared<ngraph::opset4::Add>(exp, ngraph::pattern::wrap_type<ngraph::opset4::Constant>());
+    auto log = std::make_shared<ngraph::opset4::Log>(add);
+
+    ngraph::graph_rewrite_callback matcher_pass_callback = [=](ngraph::pattern::Matcher& m) {
+        auto & pattern_to_output = m.get_pattern_value_map();
+        auto exp_input = pattern_to_output.at(input);
+
+        auto softplus = std::make_shared<ngraph::opset4::SoftPlus>(exp_input);
+
```

Comment:
Need to check that the constant input to `add` is a constant with value 1. If it is not true then we cannot apply the transformation

Reply:
Take a look at the swish_fusing.cpp for the example of how it is implemented.

Reply:
Done.